custom_dc/load_data.sh

```diff
@@ -223,9 +219,45 @@ function generate_embeddings {
   echo_log "Building embeddings for sentences in $NL_DIR"
   local cwd="$PWD"
   cd "$WEBSITE_DIR"
-  # TODO: Enable with new build_embeddings.py
-  # run_cmd python -m tools.nl.embeddings.build_custom_dc_embeddings \
-  #    --input_file_path="$NL_DIR/sentences.csv" --output_dir="$NL_DIR"
+
+  NL_EMBEDDINGS_DIR="$NL_DIR/embeddings"
+  EMBEDDINGS_PATH="$NL_EMBEDDINGS_DIR/embeddings.csv"
+  CUSTOM_EMBEDDING_INDEX="user_all_minilm_mem"
+  CUSTOM_MODEL="ft-final-v20230717230459-all-MiniLM-L6-v2"
+  CUSTOM_MODEL_PATH="gs://datcom-nl-models/ft_final_v20230717230459.all-MiniLM-L6-v2"
+  CUSTOM_CATALOG_DICT=$(cat <<EOF
+{
+  "version": "1",
+  "indexes": {
+    "$CUSTOM_EMBEDDING_INDEX": {
+      "store_type": "MEMORY",
+      "source_path": "$NL_DIR",
+      "embeddings_path": "$EMBEDDINGS_PATH",
+      "model": "$CUSTOM_MODEL"
+    }
+  },
+  "models": {
+    "$CUSTOM_MODEL": {
+      "type": "LOCAL",
+      "usage": "EMBEDDINGS",
+      "gcs_folder": "$CUSTOM_MODEL_PATH",
+      "score_threshold": 0.5
+    }
+  }
+}
+EOF
+)
+  local start_ts=$(date +%s)
+  set -x
+  python -m tools.nl.embeddings.build_embeddings \
+    --embeddings_name "$CUSTOM_EMBEDDING_INDEX" \
+    --output_dir "$NL_EMBEDDINGS_DIR" \
+    --catalog "$CUSTOM_CATALOG_DICT" >> $LOG 2>&1
+  set +x
+  status=$?
```

Comment:
move this before set, right after the python command so we get the status of python (not set).

Reply:
Thanks - done.